angular/projects/public-nrpti/src/assets/styles/themes/default.scss

```diff
@@ -29,6 +29,7 @@ $font-gray:                     #666;
 
 // Form Controls
 $fc-height:                     2.75rem;
+$fc-line-height:                1.5rem;
```

Comment:
We need to double check that this actually works across the entire site.

Reply:
It should.  We only use this for form control elements, which were originally this size anyways.  I just made it a variable so it could be used in other scss for elements that displayed alongside form control items, and needed to match styles.